### PR TITLE
[feature/SYTO-774] Create gallery variation of story card

### PR DESCRIPTION
**Pull request by bpearsoncse**
https://jira.natgeo.com/browse/SYTO-774

### CODE REVIEW

**Comment by bpearsoncse** [comment]
changes made - passing back to you @pgurley

File: app/modules/promocard/components/types/Gallery.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+import React, { PropTypes }  from 'react';
+import PromoImage from '../shared/PromoImage.jsx';
+import PromoText from '../shared/PromoText.jsx';
+import events from '../../scripts/events';
+import { generateHref } from '../../scripts/generateHref.js';
+
+const Gallery = (props) => {
+
+  const promoData = {
```
Comment (pgurley) [inline app/modules/promocard/components/types/Gallery.jsx]: Can you explain what's going on here? I don't understand why we're passing an empty object. Is this stubbed for future functionality?

**Comment by pgurley** [comment]
@bpearsoncse Mostly looks good. Had one question and for me the unit tests are failing. Are they passing for you?

```
ERROR: 'Error on Module: PromoCard', '
 Instance Id:8ad2936c-c43f-4c1b-47a6-453869f694fe', '
 Message: null is not an object (evaluating 'this.options.id')', '', '
 Stack:', 'init@http://localhost:9876/absolute/var/folders/rp/vcy86b890c52c_zwvc36r0sc0000gn/T/37258a87c63cdf8731f5b4f28231e9d3.browserify?de693dccf21b50638627dc8a57013bc534b7a3e2:1849:25
init@[native code]
http://localhost:9876/absolute/var/folders/rp/vcy86b890c52c_zwvc36r0sc0000gn/T/37258a87c63cdf8731f5b4f28231e9d3.browserify?de693dccf21b50638627dc8a57013bc534b7a3e2:3773:17
forEach@[native code]
init@http://localhost:9876/absolute/var/folders/rp/vcy86b890c52c_zwvc36r0sc0000gn/T/37258a87c63cdf8731f5b4f28231e9d3.browserify?de693dccf21b50638627dc8a57013bc534b7a3e2:3762:44
init@http://localhost:9876/absolute/var/folders/rp/vcy86b890c52c_zwvc36r0sc0000gn/T/37258a87c63cdf8731f5b4f28231e9d3.browserify?de693dccf21b50638627dc8a57013bc534b7a3e2:3449:30
http://localhost:9876/absolute/var/folders/rp/vcy86b890c52c_zwvc36r0sc0000gn/T/37258a87c63cdf8731f5b4f28231e9d3.browserify?de693dccf21b50638627dc8a57013bc534b7a3e2:77874:34
callFn@http://localhost:9876/base/node_modules/mocha/mocha.js?91557a9e8a222ad90c0392709844a7c448515b9d:4471:25
run@http://localhost:9876/base/node_modules/mocha/mocha.js?91557a9e8a222ad90c0392709844a7c448515b9d:4464:13
next@http://localhost:9876/base/node_modules/mocha/mocha.js?91557a9e8a222ad90c0392709844a7c448515b9d:4810:13
http://localhost:9876/base/node_modules/mocha/mocha.js?91557a9e8a222ad90c0392709844a7c448515b9d:4832:9
timeslice@http://localhost:9876/base/node_modules/mocha/mocha.js?91557a9e8a222ad90c0392709844a7c448515b9d:75:27'
```

File: app/modules/promocard/components/types/Gallery.jsx (hunk 1 above)
Comment (bpearsoncse) [inline app/modules/promocard/components/types/Gallery.jsx]: Yes, it was on some previous AC to be able to accept an onClick function. Not sure if that is still needed now, as we have a lot of other onClick functionality going on. I left it in for now, but I'm open to removing it.

**Comment by pgurley** [comment]
Merged

**Pull request merged**